.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,32 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - deploy-try-vyper
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: "Deploy to try.vyperlang.org"
+
+    steps:
+      - name: Configure SSH
+        run: |
+          mkdir -p ~/.ssh/
+          echo "${{ secrets.TRY_VYPER_KEY }}" > ~/.ssh/try_vyper.key
+          chmod 600 ~/.ssh/try_vyper.key
+          cat >>~/.ssh/config <<END
+          Host try_vyper
+            HostName try.vyperlang.org
+            User ${{ secrets.TRY_VYPER_USERNAME }}
+            IdentityFile ~/.ssh/try_vyper.key
+            StrictHostKeyChecking no
+          END
+
+      - name: Deploy
+        run: |
+          ssh try_vyper bash <<EOF
+          cd try.vyperlang.org
+          REF=${{ github.sha }} ./full_reload.sh
```

Comment:
is the github sha needed? i think full_reload.sh should grab latest boa no?

Reply:
By using the sha we can keep the docker cache

Reply:
Basically it caches these lines: https://github.com/vyperlang/try.vyperlang.org/blob/c6148d8d66fe1a20a9883edb6de668cb2619f03b/notebook.Dockerfile#L4-L6

Reply:
no more cache :smiling_face_with_tear: